actions/install/action.yml

```diff
@@ -0,0 +1,19 @@
+name: setup
+description: 'Installs SLSA verifier and adds it to your PATH'
+branding:
+  icon: 'package'
+  color: 'blue'
+inputs:
+  verifier-release:
```

Comment:
shall we call this `verifier-version`? GitHub actions use `python-version` for Python and `node-version` for node, so this may be a more standard naming convention...?